views/backend/settings.php

```diff
@@ -69,7 +77,7 @@
                             <div class="card-content white-text">
                                 <?php
                                 foreach($errors as $error){
-                                    echo $error."<br/>";
+                                    echo $error;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$error'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)